packages/backend-modules/mail/templates/payment_successful_abo_give.html

```diff
@@ -91,7 +91,11 @@
                       2022 verschickt. {{/if}}{{#if goodies_has_mug}}Republik-Tassen 
                       liefern wir Ihnen innerhalb von 10 Werktagen. {{/if}} 
                       {{#if goodies_has_cookie}} Der Versand der Republik-Guetsliformen 
-                      erfolgt bis 22. Dezember. {{/if}}{{#if goodies_has_totebag_black}}Schwarze Taschen liefern wir im April. {{/if}}
+                      erfolgt bis 22. Dezember. {{/if}}{{#if goodies_has_totebag_black}}
+                      Schwarze Taschen liefern wir im April. {{/if}}
+                      {{#if goodies_has_librarybook}} Republik-Bibliothek Taschenbüber, 
```

Comment:
I was wondering why there's no change to the backend needed, in order to set this variable in the template. For reference, it looks like this is automagically handled in https://github.com/republik/plattform/blob/53fd4ea82c3127ac738fd2b3cdbd409de3909467/packages/backend-modules/republik-crowdfundings/lib/Mail.js#L794-L796

🥲